source/javascripts/components/_Variable.js.erb

```diff
@@ -83,6 +90,18 @@ angular.module("BitriseWorkflowEditor").factory("Variable", function(Meta) {
 		return true;
 	};
 
+	Variable.isValidValue = function(value, shouldBeNotEmpty) {
+		if (value === undefined) {
+			return undefined;
+		}
+
+		if (shouldBeNotEmpty && value !== null && value.replace(new RegExp(/^(\s)*$/gm), '').length < 1) {
```

Comment:
No need to replace, we could do a simple match here, regexp.match(value), regexp being the same as the current one - except that we do not need to capture the spaces, so `^(?:\s)*$` would be better.